builtin/logical/consul/path_roles.go

```diff
@@ -89,6 +89,18 @@ created within. Defaults to 'default'. Available in Consul 1.7 and above.`,
 				Description: `Indicates which admin partition that the token
 will be created within. Defaults to 'default'. Available in Consul 1.11 and above.`,
 			},
+
+			"service_identities": {
+				Type: framework.TypeString,
```

Comment:
Using `TypeString` with semicolon delimited service identities seems like it will work, but an alternative to consider would be `TypeStringSlice`. The comma splitting doesn't happen automatically by the framework using the type. 

Supplying the service identities would look something like:

```go
vault write consul/roles/serv-id-role \
    service_identities="webserv-1:dc1,dc2,dc3" \
    service_identities="webserv-2" \
    service_identities="webserv-3:dc1"
```

It might simplify the parsing a bit too. Let me know your thoughts 👍

Reply:
This was a great idea as it eliminated the second string split.

Additionally, this matches the user experience of Consul itself, where you can specify `-service-identity=...` multiple times.